exit with build status

shanmukhananda opened the pull request:
changes to address https://github.com/paul-michalik/ORB_SLAM2/issues/83

paul-michalik commented on scripts/linux/ci_cd.sh:
```diff
@@ -41,3 +41,5 @@ then
     fi
 fi
+
+exit ${build_status}
```
Exit early instead of keeping the "status" of all possible failures. Not being a bash expert, I guess you can enforce that mode for a script with `set -e`.

paul-michalik commented:
We should make this a law for all scripts. And for all kinds of code actually ;-)

shanmukhananda commented:
added `set -e` to all `.sh` files. Also I removed clean up from `bootstrap.sh` cleanup will be done from `Dockerfile`. I have commented uploading of docker images from CI setup, as Travis.CI has time limits. we can use this code if auto upload of docker images is required in future.
